App gallery (#25)
windows-toolkit/WindowsCommunityToolkit-wiki: Home.md

- Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
- Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
- We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!

## Projects using the Windows Community Toolkit

Below are a few projects highlighting usage of the Windows Community Toolkit from other creators in our Community. We hope they inspire you to see the possibilities of what you're able to create with the tools provided in our library.

### [Fiona](https://www.mysqueezebox.com/download) by [PaoloM](https://github.com/PaoloM)

> ![Fiona1](/images/App-Gallery/Fiona1.png)

Reviewer: These are pretty giant images still, can we convert them to smaller jpgs?

- Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
- Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
- We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!

## Projects using the Windows Community Toolkit

Below are a few projects highlighting usage of the Windows Community Toolkit from other creators in our Community. We hope they inspire you to see the possibilities of what you're able to create with the tools provided in our library.

### [Fiona](https://www.mysqueezebox.com/download) by [PaoloM](https://github.com/PaoloM)

> ![Fiona1](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png)

Reviewer: Let's clean-up the images below in the image folder as a sub-folder, and for only the one's we're using (images/app-gallery), then we can reference them relatively like:

`![Expanded PR Checks with Build Number](images/Preview-Packages-PR-Checks.png)`

Let's stick with pngs or jpgs vs large gifs as we don't want a ton of images in the git repo.

> Fiona was started using the WCT Template, and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.

Author: ```suggestion
Fiona was started using the WCT Template ([from Windows Template Studio](https://aka.ms/wts)), and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.
```

- Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
- Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
- We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!

> ## Projects that are currently using Windows Community Toolkit...

Reviewer: ```suggestion
## Projects using the Windows Community Toolkit
```

We may want a blurb introducing this under the header? Like:

> Below are a few projects highlighting usage of the Windows Community Toolkit from other creators in our Community. We hope they inspire you to see the possibilities of what you're able to create with the tools provided in our library.

> <h1 align="center">Fiona</h1>

Reviewer: ```suggestion
### Fiona
```

<table>
  <tr>

>     <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png height=350></td>

Reviewer 2: I am not sure these links are a good idea, we should use images from the `WindowsCommunityToolkit`'s repo. Maybe we can use a relative path to reference them.

Reviewer: Let's also have 1 image per app, think the first one for Fiona is the best representation.

    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona2.png height=350></td>
  </tr>
  <tr>
    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona4.png height=350></td>
    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona3.png height=350></td>
  </tr>
</table>

>Creator: [PaoloM](https://github.com/PaoloM)

> <p align="left">Fiona was started using the WCT Template, and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.</p>

Reviewer: ```suggestion
Fiona was started using the WCT Template ([from Windows Template Studio](https://aka.ms/wts)), and it's using a bunch of the helpers and services provided with that. Plus, as mentioned, the toolkit MVVM.
```
We can probably switch the HTML to Markdown here for the headers and things to keep it a bit more simple.

 
 > You can find out more information about **Fiona** [here](https://github.com/PaoloM/Fiona)

> <h1 align="center">Shrestha Files Pro</h1>

Reviewer: ```suggestion
### Shrestha Files Pro
```

<h1 align="center">Yugen Mosaic</h1>

![YugenMosaic](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic.png) 
![YugenMosaic2](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic2.png)

>Creator: [PandaSharp](https://github.com/Panda-Sharp)

Yugen Mosaic is a free app that creates digital art pictures based on your own images. With it, you can use an unlimited amount of images, to create a photo mosaic which is an image composed of many tiled photos. Yugen is an awareness of the universe that triggers emotional responses too deep and powerful for words, and that is just what your art will do with a little help from Yugen Mosaic!

Find the App in [Microsoft Store](https://www.microsoft.com/en-us/p/yugen-mosaic/9pf0s24cx0d4). More information regarding this [project](https://github.com/Panda-Sharp/Yugen.Mosaic)

<h1 align="center">OneLocker and Legere</h1>

![Onelocker-legere](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Onelocker-Legere.png)

> >Creator: [Sergio Pedri](https://github.com/Sergio0694)

Reviewer: @Sergio0694 which one would you want to highlight here? Pick one. 😉

Reviewer 3: I'm confused - is the issue just that the image has two apps together, or do we want to limit the gallery to one app per developer? Because if it's the latter then I guess I'd have to also remove my console/IDE further down in the page as well? Let me know, and if needed I can provide alternative assets as well or anything else 🙂

Author: @Sergio0694 As of now we would like to have one image so we don't clutter the page too much.

Author: @Sergio0694 let me know whichever project you want to be featured in the Gallery and please provide Github/project link of that as well. Thanks 😊

Reviewer 3: Hey @Kyaa-dost, sorry for the delay! Got busy working on a PR for .NET 6 to support the MVVM Toolkit 😄

Between the two I guess I'd prefer to feature Legere, as it's the most recent app and it uses the Toolkit the most.
Here's an image just with that: [Legere](https://user-images.githubusercontent.com/10199417/122304755-05dc2900-cf06-11eb-96ab-1681b36677bb.png).
Here's the Store link: https://www.microsoft.com/store/apps/9PHJRVCSKVJZ.

And just in case you decided to allow more than one app per dev, then the second one for me would be my Console/IDE, which is also open source on GitHub, and heavily uses a lot of stuff from the Toolkit, especially .NET APIs such as the HighPerformance and Diagnostics packages and the MVVM Toolkit 🚀
Here's an updated image for that: [console/IDE](https://user-images.githubusercontent.com/10199417/122305064-869b2500-cf06-11eb-8c6a-d5b7b04eb614.png).
And here's the GitHub link: https://github.com/Sergio0694/Brainf_ckSharp.

Let me know if this is alright!

Reviewer: Yeah, for now just wanted to limit one-per dev. I think eventually we'll try and better integrate with the UWP Community site, we're just kind of doing this as a temporary measure in the meantime.

Reviewer 3: I see, sounds good! 😄

**OneLocker:** is a UWP password manager with the goal of being as feature complete as possible. It supports tons of UWP-specific features (eg. Windows Hello, Timeline, live tiles, notifications, background sync, Windows Ink, Surface Dial, etc.) and the database is heavily customizable. The creator have used the Toolkit here for several UWP helpers (eg. SystemInfo) and extensions.

**Legere:** is a fully customizable UWP client for Reddit supporting multiple users, live tiles, myTube! integration, and more. 

**According to the Creator:**

> ***I'm using several Toolkit controls like the WrapPanel to display post tags, as well as lots of helpers. I'm currently in the process of migrating the codebase to the Toolkit 7.0***

<h1 align="center">Console/IDE</h1>

![Console/IDE](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Console:IDE.png)

>Creator: [Sergio Pedri](https://github.com/Sergio0694)

Console/IDE: is a fully featured console and IDE/debugger for an obscure programming language.

> **In the words of Sergio**: 

Reviewer: We should be consistent about how we quote the creators for each app. So we should be like:

```
### [App Name](Link to store) by [Creator](Link to GitHub Profile)
(Image) + link to Store/GitHub
App Description
> "Quote from Developer" - _Developer Name_
Toolkit Components Used: MVVM Toolkit, WrapPanel, etc...
[GitHub](link) - [Store](link)
```
